docs/version-history/feature-versions-23-x.rst

```diff
@@ -43,6 +43,10 @@ New Features:
   cause problems with code that tried to read the whole scratch directory.
   :jira:`2564`
 
+- Docker Universe jobs now report memory usage when running on docker systems
+  that sit on top of cgroup v2 systems.
+  :jira:`2573`
+
```

Comment:
```suggestion
- Rather than report no memory usage, Docker universe jobs now over-report memory usage
  (by including memory used for caching) when running on modern kernels.
  :jira:`2573`
```
?